pages/common/jj-interdiff.md

```diff
@@ -0,0 +1,32 @@
+# jj interdiff
+
+> Compare changes of two revisions.
+> More information: <https://jj-vcs.github.io/jj/latest/cli-reference/#jj-interdiff>.
+
+- Compare changes from a revision to the working copy:
+
+`jj interdiff {{[-f|--from]}} {{revset}}`
```

Comment:
```suggestion
`jj interdiff {{[-f|--from]}} {{revision_set}}`
```
Could you please update this to improve readability?

Reply:
Let it be until we jmhave all the pages docunebted. Then standardize them all in one swoop